app/controllers/Application.java

```diff
@@ -433,22 +433,26 @@ public static Result compare() {
 
     List<AppResult> results1 = null;
     List<AppResult> results2 = null;
-    if (partialFlowExecId1 != null && !partialFlowExecId1.isEmpty() && partialFlowExecId2 != null && !partialFlowExecId2.isEmpty()) {
-      IdUrlPair flowExecIdPair1 = bestSchedulerInfoMatchGivenPartialId(partialFlowExecId1, AppResult.TABLE.FLOW_EXEC_ID);
-      IdUrlPair flowExecIdPair2 = bestSchedulerInfoMatchGivenPartialId(partialFlowExecId2, AppResult.TABLE.FLOW_EXEC_ID);
-      results1 = AppResult.find
-          .select(AppResult.getSearchFields() + "," + AppResult.TABLE.JOB_DEF_ID + "," + AppResult.TABLE.JOB_DEF_URL
-              + "," + AppResult.TABLE.FLOW_EXEC_ID + "," + AppResult.TABLE.FLOW_EXEC_URL)
-          .where().eq(AppResult.TABLE.FLOW_EXEC_ID, flowExecIdPair1.getId()).setMaxRows(100)
-          .fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS, AppHeuristicResult.getSearchFields())
-          .findList();
-      results2 = AppResult.find
-          .select(
-              AppResult.getSearchFields() + "," + AppResult.TABLE.JOB_DEF_ID + "," + AppResult.TABLE.JOB_DEF_URL + ","
-                  + AppResult.TABLE.FLOW_EXEC_ID + "," + AppResult.TABLE.FLOW_EXEC_URL)
-          .where().eq(AppResult.TABLE.FLOW_EXEC_ID, flowExecIdPair2.getId()).setMaxRows(100)
-          .fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS, AppHeuristicResult.getSearchFields())
-          .findList();
+    if (partialFlowExecId1 != null && !partialFlowExecId1.isEmpty() && partialFlowExecId2 != null
```

Comment:
Because this PR is already huge, formatting of unrelated lines will make debugging harder in the future. Please revert this file's formatting back to how it was. It can be addressed in a separate PR.

Reply:
Not possible to revert it back unless revert it back to original and then apply my changes again. Almost all other changes are formatting changes other than new getCurrentRunParameters method.

Reply:
Reverting should be simple. You can revert a file selectively.
```
1. git checkout <path-to-this-file> which will revert it to the original.
2. Add the getCurrentRunParameters method.
```

Otherwise `git blame` will be get screwed up and be useless in the future.